src/Components/Calculator/CalculatorButton/CalculatorKey.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from "react";
+import './CalculatorKey.css'
+
+interface Props {
+    keyValue: number | string;
+    className: string;
+    isDisable?: boolean;
+    onKeyClick: (key: number | string) => void
+}
+
+const CalculatorKey: React.FC<Props> = (props) => {
+
+    return (
+        <button className={props.className} onClick={() => props.onKeyClick(props.keyValue)} disabled={props.isDisable}>
```

Comment:
Why dont you use object destructuring here?
Like 
`const {className, keyValue, isDisable} = props`
`<button className={className} onClick=() => .,....`